client/blocks/reader-site-notification-settings/index.jsx

```diff
@@ -132,7 +136,7 @@ class ReaderSiteNotificationSettings extends Component {
 					<Gridicon icon="cog" size={ 24 } ref={ this.saveIconRef } />
 					<span
 						className="reader-site-notification-settings__button-label"
-						title={ translate( 'Email settings' ) }
+						title={ translate( 'Notification settings' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'notification settings' )` *ES Score: 20*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/reader/show-settings-popover-even-if-emails-blocked)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).